### DISCUSSION TITLE
File types for Merging 10X multiome datasets

### DISCUSSION THREAD

**Discussion by sylestiel** [stuart-lab/signac#1146] [Q&A] [answered]
Glancing at the Merge objects vignette I'm not clear on the files and steps needed for merger of a multiome dataset. 
When analyzing a single multiome dataset we read in the counts (filtered_feature_bc_matrix.h5) and the fragment files (fragment.tsv.gz) for each of them.
However, the Merging Objects vignette (which is for scATAC datasets) outlines generation of a common peak set and then a fragment object using peak.bed and singlecell.csv files and then proceeding with  Chromatin Assay object creation .
So what files and steps do I use for merging two Multiome datasets?
Do we work from the peaks.bed and singlecell.csv instead of the fragment.tsv.gz file? What about the gene expression component?

**Comment by timoast** [comment, marked as answer]
You can follow exactly the same steps as is shown in the scATAC-seq [merge vignette](https://satijalab.org/signac/articles/merging.html), the only difference is each object with have another assay for the RNA.
